docs/how-to/how-to-use-erofs-build-rootfs.md

```diff
@@ -0,0 +1,100 @@
+# Configure Kata Containers to use EROFS build rootfs
+
+## Introduction
+For kata containers, rootfs is used in the read-only way. EROFS can noticeably decrease metadata overhead.
+
+`mkfs.erofs` can generate compressed and uncompressed EROFS images.
+
+For uncompressed images, no files are compressed. However, it is optional to inline the data blocks at the end of the file with the metadata.
+
+For compressed images, each file will be compressed using the lz4 or lz4hc algorithm, and it will be confirmed whether it can save space. Use No compression of the file if compression does not save space.
+
+## Performance comparison
+|                 | EROFS | EXT4 | XFS |
+|-----------------| --- | --- | --- |
+| Image Size [MB] | 106 | 256 | 126 |
```

Comment:
Compressed Image size (obvious compressed EROFS is smaller than 106MiB.)

Reply:
I noted that this is the uncompressed image size.